teuthology/orchestra/remote.py

```diff
@@ -206,6 +206,22 @@ def run(self, **kwargs):
         r.remote = self
         return r
 
+    def mkdtemp(self, suffix=None, parentdir=None):
+        """
+        Create a temporary directory on remote machine and return it's path.
+        """
+        args = ['mktemp', '-d']
+
+        if suffix:
+            args.append('--suffix')
+            args.append(suffix)
+
+        if parentdir:
+            args.append('-p')
+            args.append(parentdir)
+
+        return self.sh(args=args, stdout=StringIO()).strip()
```

Comment:
- you need to provide args as a first argument.
- you do not need to provide stdout the _sh()_ it is already done by default
so:
```
return self.sh(args).strip()
```

Reply:
not `strip()`ing since I am adding that call to sh().